packages/wdio-appium-service/src/launcher.ts

```diff
@@ -166,25 +170,54 @@ export default class AppiumLauncher implements Services.ServiceInstance {
                     log.info(`Appium started with ID: ${process.pid}`)
                     resolve(process)
                 }
+                // if the data does not include the above string, it is probably an error or debugger message.
+                log.warn(data.toString())
+                resolve('The appium service is starting... Check logs above for possible issues.')
```

Comment:
We resolve the promise with a string here which would cause an error further up where a child process is expected.

Reply:
Interesting, so we can resolve with the same promise on this case so the appium process finished well but without expect message. 

A quick change if you are agree.